File: cfgs/java_cfgs_foo/MINIMUM_INCREMENT_K_OPERATIONS_MAKE_ELEMENTS_EQUAL.java
Nodes
1;special;;;
2;statement;fun_1(foo_1);14;
3;statement;int max = foo_1[foo_2.length - 1];15;
4;statement;int res = 0;16;
5;statement;int i = 0;17;
6;conditional;foo_1 < foo_2;18;
7;conditional;((foo_1 - foo_2[foo_3]) % foo_4) != 0;20;
8;statement;return -1;20;
9;exit;;;
10;statement;res += (foo_1 - foo_2[foo_3]) / foo_4;21;
11;statement;i++;19;
12;statement;return foo_1;23;
Edges
0;2;
1;3;
2;4;
3;5;
4;6;
5;7;12;
6;8;10;
7;9;
8;
9;11;
10;6;
11;9;

File: cfgs/pythoncfgs/MINIMUM_INCREMENT_K_OPERATIONS_MAKE_ELEMENTS_EQUAL.py
Nodes
1;special;;;
2;statement;foo_1 = fun_1(foo_2);8;
3;statement;foo_1 = 0;9;
4;conditional;foo < foo;10;
5;statement;return fun_1(foo_1);15;
6;conditional;(foo_1 - foo_2[foo_3]) % foo_4 != 0;11;
7;statement;foo_1 += (foo_2 - foo_3[foo_4]) / foo_5;14;
8;statement;return -1;12;
9;exit;;
Edges
0;2;
1;3;
2;4;
3;5;6;
4;9;
5;7;8;
6;4;
7;9;
8;